### ISSUE TITLE
Size::fitsInto does the opposite of what it should

### ISSUE THREAD

**Issue by tocsick** [Intervention/image#320]
The function Size::fitsInto actually does the exact opposite of what it should do which has some bump on effects on functions like Image::fit

It should be:
return ($this->width >= $size->width) && ($this->height >= $size->height);

Not:
return ($this->width <= $size->width) && ($this->height <= $size->height);

**Comment by olivervogel** [comment]
Yes you're right, the comment should be `Checks if current size fits into given size`.

```php
$box = new Size(800, 600);
$fits = $box->fitsInto(new Size(100, 100));
$this->assertFalse($fits);
````

```php
$box = new Size(800, 600);
$fits = $box->fitsInto(new Size(1000, 1000));
$this->assertTrue($fits);
```
